kie-dmn/kie-dmn-validation/src/main/java/org/kie/dmn/validation/DMNValidatorImpl.java

```diff
@@ -98,6 +115,10 @@ public DMNValidatorImpl() {
             String message = MsgUtil.createMessage( Msg.FAILED_VALIDATOR );
             failedInitMsg.add(new DMNMessageImpl(DMNMessage.Severity.ERROR, message, Msg.FAILED_VALIDATOR.getType(), null ) );
         }
+        ChainedProperties localChainedProperties = new ChainedProperties();
+        this.dmnProfiles.addAll(DMNAssemblerService.getDefaultDMNProfiles(localChainedProperties));
+        this.dmnProfiles.addAll(dmnProfiles);
+        this.dmnCompilerConfig = DMNAssemblerService.compilerConfigWithKModulePrefs(localChainedProperties, this.dmnProfiles);
```

Comment:
the compiler configuration, used below when instantiating the compiler, is defined by:
- checking local ChainedProperties (at the moment we don't support user providing this externally)
- any default profiles, as descending from the refactored code on DMNAssembler
- any user-supplied profiles